ci-operator/step-registry/cucushift/installer/check/ibmcloud/private/cucushift-installer-check-ibmcloud-private-commands.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output
+    region="${LEASED_RESOURCE}"
+    rg=$1
+    export region
+    echo "Try to login to ${rg}..."
+    "${IBMCLOUD_CLI}" login -r ${region} -g ${rg} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+RESOURCE_GROUP=$(jq -r .ibmcloud.resourceGroupName ${SHARED_DIR}/metadata.json)
+
+ibmcloud_login ${RESOURCE_GROUP}
+
+critical_check_result=0
+
+#check the load balances 'Is public' all are false
+mapfile -t lbs < <(ibmcloud is lbs --output JSON | jq -r '.[]|.name+" "+(.is_public|tostring)')
+echo "INFO: lb list is: " "${lbs[@]}"
+for lb in "${lbs[@]}"
+do
+    if [[ ! ${lb} =~ false ]]; then
+        echo "ERROR: $lb is not expected value!"
+        critical_check_result=1
+    fi
+done
+
+#check the instances without floating IP (Public IP)
+mapfile -t ins < <(ibmcloud is instances -q | awk 'NR>1 {print $2 "|" $5}')
```

Comment:
I do not think you need to check instance FIP, because even it is common ipi install, all the masters and workers have no public IP. The only difference between private and non-private is bootstrap, but in this case, we do not have bootstrap kept.